src/runtime/meter-toast-api.js

```diff
@@ -33,8 +33,10 @@ export const MINIMIZED_IFRAME_SIZE = '420px';
 export class MeterToastApi {
   /**
    * @param {!./deps.DepsDef} deps
+   * @param {string=} iframeUrl Relative URL of the iframe, e.g. "/meteriframe".
+   * @param {Object<string, string>=} iframeParams List of extra params appended to the URL.
    */
-  constructor(deps) {
+  constructor(deps, iframeUrl = '/metertoastiframe', iframeParams = {}) {
```

Comment:
Hmmm can we maybe make an interface for these additional params to fit into? I never loved the idea of adding more than a few params to functions when deconstructing is so easy in JS. @ChrisAntaki any thoughts?

Reply:
Good idea. Done.